nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/output.md

```diff
@@ -15,16 +15,21 @@ and processes data using the following steps:
 
 For further reading and documentation see the [FastQC help](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
 
+> **NB:** The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality. To see how your reads look after trimming, look at the FastQC reports in the `trim_galore` directory.
+
 **Output directory: `results/fastqc`**
 
 * `sample_fastqc.html`
   * FastQC report, containing quality metrics for your untrimmed raw fastq files
-* `zips/sample_fastqc.zip`
+* `sample_fastqc.zip`
```

Comment:
Nope, the template does put these into a subfolder. See: https://github.com/nf-core/tools/blob/1fa5ec80f797ac03d9f72e2228e9d0abe5e6bb12/nf_core/pipeline-template/%7B%7Bcookiecutter.pipeline_slug%7D%7D/main.nf#L205-L208